README.md

```diff
@@ -170,6 +170,7 @@ Asynchronous concurrent distributed networking framework for PHP
 
 * [amphp/mysql](https://github.com/amphp/mysql) - An async mysql client built on [amp](#amphp)
 * [amphp/pgsql](https://github.com/amphp/pgsql) - A non-blocking PostgreSQL library built on [amp](#amphp)
+* [amphp/redis](https://github.com/amphp/redis) - An async redis client built on [amp](#amphp)
```

Comment:
I'm not sure where to list it. You have other things in multiple places as well, so I did this here as well.